django-drf-react-quickstart/mcbcapp/frontend/src/components/IntroPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+//import logo from "../assets/icons/logo.png";
+
+//console.log(logo);
+
+class IntroPage extends Component{ 
+    render(){
+               
+            const backgroundStyle = {
+                backgroundColor: "#32669d", 
+                width: "1919.9px", 
+                height: "1080px"
+            };
+            const nameStyle1 = {
+                color: "white", 
+                fontSize:"50px",  
+                position: "center" 
+            };
+            const nameStyle2 = {
+                color: "white", 
+                fontSize:"20px",  
+                position: "center" 
+            };
+
+            return(
+                <body style={backgroundStyle}>
+                
+                <center>
+                    <br></br><br></br><br></br><br></br><br></br><br></br><br></br>
```

Comment:
lmao don't use line breaks - even if its centred on your own screen, if you view it on different screen sizes it won't be centred
consider wrapping your icon + text in a <div> and styling it, perhaps with align-items : center or something like that

Reply:
Can't implement align items or an alternative to solve this issue